### DISCUSSION TITLE
Can't install llama-cpp-python from requirement text

### DISCUSSION THREAD

**Discussion by AAbdullah1918** [imartinez/privateGPT#948] [Q&A]
I want to use eachadea/ggml-vicuna-13b-1.1 as my LLM model. I made the appropriate changes to the .env file but when I run the pip3 install -r requirements.txt command in cmd, it gives me the following error:
`  Building wheel for llama-cpp-python (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for llama-cpp-python (pyproject.toml) did not run successfully.`

`  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for llama-cpp-python
  Building wheel for hnswlib (pyproject.toml) ... error
  error: subprocess-exited-with-error`

` Building wheel for hnswlib (pyproject.toml) did not run successfully.`
`Failed to build llama-cpp-python hnswlib
ERROR: Could not build wheels for llama-cpp-python, hnswlib, which is required to install pyproject.toml-based projects`

I was following this tutorial on how to install PrivateGPT, if that helps: [https://www.youtube.com/watch?v=A3F5riM5BNE&list=PLDlsxP8Mhcy3Gq9bNDhNthE6h4fzTa1YE&index=1](url)

How could I resolve this error? Any help is appreciated. Thanks

**Comment by Jawn78** [comment]
```
pip uninstall llama-cpp-python
export CMAKE_ARGS="-DLLAMA_CUBLAS=on"
export FORCE_CMAKE=1
pip install llama-cpp-python --no-cache-dir
```
